README.md

```diff
@@ -1,9 +1,21 @@
-# Apostrophe 3 demo and test project
+# Apostrophe 3 essentials starter kit
 
-## Get started
+## Getting started
 
-If you created your project using the CLI tool, all dependencies will already have been installed, so you can skip this step. Otherwise, install 
-the dependencies using: `npm install`
+This Starter Kit serves as a template for initiating new projects and is intended for use in two main ways:
```

Comment:
We also discussed adding "also known as a boilerplate project..." language here for both pedagogical and SEO reasons, it's a thing people look for.

Reply:
Changed